src/org/opensolaris/opengrok/web/Util.java

```diff
@@ -1294,7 +1294,7 @@ public static String linkify(String url, boolean newTab) {
                 return String.format("<a href=\"%s\"%s title=\"Link to %s\">%s</a>",
                         constructed.toString(),
                         newTab ? " target=\"_blank\"" : "",
-                        Util.htmlize(url),
+                        Util.encode(url),
```

Comment:
The comment next to `Util.encode()` says `Currently used only for tooltips of annotation revision number view` so it should be augmented.